.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: Create a report to help us improve
+labels: bug
+---
+
+<!-- Thanks for filing a 🐛 bug report 😄! -->
+
+**Problem**
+<!-- A clear and concise description of what the bug is. -->
+<!-- including what currently happens and what you expected to happen. -->
+
+**Steps**
+<!-- The steps to reproduce the bug. -->
+1.
+2.
+3.
+
+**Possible Solution(s)**
+<!-- Not obligatory, but suggest a fix/reason for the bug, -->
+<!-- or ideas how to implement the addition or change -->
+
+**Notes**
+
+Output of `rustup --version`:
+Output of `rustc --version`:
```

Comment:
I think rather than `rustc --version` it might be better to have `rustup show` output